nexus/templates/nexus/base.html

```diff
@@ -19,7 +19,7 @@
         <script src="{% nexus_media_prefix %}/nexus/js/lib/facebox/facebox.js"></script>
         <script src="{% nexus_media_prefix %}/nexus/js/nexus.js"
                 id="nexus-constants"
-                data-csrf-cookie-name="{% nexus_csrf_cookie_name %}"
+                data-csrf-token="{{ csrf_token }}"
```

Comment:
no need for 'nexus' in name imo, the nexus in nexus-media-prefix was pre-existing for the `nexus_media_prefix` tag